rail_id	external_id	study	sample_acc	experiment_acc	submission_acc	submission_center	submission_lab	study_title	study_abstract	study_description	experiment_title	design_description	sample_description	library_name	library_strategy	library_source	library_selection	library_layout	paired_nominal_length	paired_nominal_stdev	library_construction_protocol	platform_model	sample_attributes	experiment_attributes	spot_length	sample_name	sample_title	sample_bases	sample_spots	run_published	size	run_total_bases	run_total_spots	num_reads	num_spots	read_info	run_alias	run_center_name	run_broker_name	run_center
1475251	SRR6145415	SRP119561	SRS2570967	SRX3257506	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806214: 2-12A.RNA.Young; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Young|tissue;;lateral temporal lobe	GEO Accession;;GSM2806214		GSM2806214	2-12A.RNA.Young	2930154782	39471683	2018-03-08 00:04:45	1332846011	2930154782	39471683	2	39471683	index:0,count:39471683,average:74.23,stdev:3.23	GSM2806214_r1			
1475282	SRR6145416	SRP119561	SRS2570969	SRX3257507	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806215: 3-17T.RNA.Young; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Young|tissue;;lateral temporal lobe	GEO Accession;;GSM2806215		GSM2806215	3-17T.RNA.Young	1931673387	26043105	2018-03-08 00:04:45	876584528	1931673387	26043105	2	26043105	index:0,count:26043105,average:74.17,stdev:2.78	GSM2806215_r1			
1475315	SRR6145417	SRP119561	SRS2570968	SRX3257508	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806216: 4-13A.RNA.Young; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Young|tissue;;lateral temporal lobe	GEO Accession;;GSM2806216		GSM2806216	4-13A.RNA.Young	2650844506	35768170	2018-03-08 00:04:45	1194122280	2650844506	35768170	2	35768170	index:0,count:35768170,average:74.11,stdev:3.60	GSM2806216_r1			
1475347	SRR6145418	SRP119561	SRS2570970	SRX3257509	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806217: 5-18T.RNA.Young; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Young|tissue;;lateral temporal lobe	GEO Accession;;GSM2806217		GSM2806217	5-18T.RNA.Young	2835859250	38207241	2018-03-08 00:04:45	1278877674	2835859250	38207241	2	38207241	index:0,count:38207241,average:74.22,stdev:2.66	GSM2806217_r1			
1475380	SRR6145419	SRP119561	SRS2570972	SRX3257510	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806218: 6-14A.RNA.Young; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Young|tissue;;lateral temporal lobe	GEO Accession;;GSM2806218		GSM2806218	6-14A.RNA.Young	2861442265	38638267	2018-03-08 00:04:45	1281264766	2861442265	38638267	2	38638267	index:0,count:38638267,average:74.06,stdev:3.93	GSM2806218_r1			
1475603	SRR6145420	SRP119561	SRS2570971	SRX3257511	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806219: 7-19T.RNA.Young; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Young|tissue;;lateral temporal lobe	GEO Accession;;GSM2806219		GSM2806219	7-19T.RNA.Young	2857376367	38465281	2018-03-08 00:04:45	1301841355	2857376367	38465281	2	38465281	index:0,count:38465281,average:74.28,stdev:2.81	GSM2806219_r1			
1475636	SRR6145421	SRP119561	SRS2570973	SRX3257512	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806220: 8-15A.RNA.Young; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Young|tissue;;lateral temporal lobe	GEO Accession;;GSM2806220		GSM2806220	8-15A.RNA.Young	2273402001	30568266	2018-03-08 00:04:45	1040571074	2273402001	30568266	2	30568266	index:0,count:30568266,average:74.37,stdev:2.30	GSM2806220_r1			
1475669	SRR6145422	SRP119561	SRS2570974	SRX3257513	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806221: 9-16A.RNA.Young; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Young|tissue;;lateral temporal lobe	GEO Accession;;GSM2806221		GSM2806221	9-16A.RNA.Young	2730631320	36817470	2018-03-08 00:04:45	1235825139	2730631320	36817470	2	36817470	index:0,count:36817470,average:74.17,stdev:3.31	GSM2806221_r1			
1475702	SRR6145423	SRP119561	SRS2570975	SRX3257514	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806222: 10-8A.RNA.Old; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Old|tissue;;lateral temporal lobe	GEO Accession;;GSM2806222		GSM2806222	10-8A.RNA.Old	2785289949	37552625	2018-03-08 00:04:45	1247512662	2785289949	37552625	2	37552625	index:0,count:37552625,average:74.17,stdev:3.49	GSM2806222_r1			
1475732	SRR6145424	SRP119561	SRS2570976	SRX3257515	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806223: 11-10T.RNA.Old; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Old|tissue;;lateral temporal lobe	GEO Accession;;GSM2806223		GSM2806223	11-10T.RNA.Old	2822922825	38052606	2018-03-08 00:04:45	1277669110	2822922825	38052606	2	38052606	index:0,count:38052606,average:74.18,stdev:3.30	GSM2806223_r1			
1475766	SRR6145425	SRP119561	SRS2570978	SRX3257516	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806224: 12-6A.RNA.Old; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Old|tissue;;lateral temporal lobe	GEO Accession;;GSM2806224		GSM2806224	12-6A.RNA.Old	2685744686	36212177	2018-03-08 00:04:45	1210146374	2685744686	36212177	2	36212177	index:0,count:36212177,average:74.17,stdev:3.27	GSM2806224_r1			
1475796	SRR6145426	SRP119561	SRS2570977	SRX3257517	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806225: 13-11T.RNA.Old; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Old|tissue;;lateral temporal lobe	GEO Accession;;GSM2806225		GSM2806225	13-11T.RNA.Old	3118832602	42174836	2018-03-08 00:04:45	1416524404	3118832602	42174836	2	42174836	index:0,count:42174836,average:73.95,stdev:3.93	GSM2806225_r1			
1475830	SRR6145427	SRP119561	SRS2570979	SRX3257518	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806226: 14-7A.RNA.Old; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Old|tissue;;lateral temporal lobe	GEO Accession;;GSM2806226		GSM2806226	14-7A.RNA.Old	2860577582	38610276	2018-03-08 00:04:45	1277270901	2860577582	38610276	2	38610276	index:0,count:38610276,average:74.09,stdev:3.62	GSM2806226_r1			
1475861	SRR6145428	SRP119561	SRS2570980	SRX3257519	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806227: 15-13T.RNA.Old; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Old|tissue;;lateral temporal lobe	GEO Accession;;GSM2806227		GSM2806227	15-13T.RNA.Old	2989330940	40295518	2018-03-08 00:04:45	1346658663	2989330940	40295518	2	40295518	index:0,count:40295518,average:74.19,stdev:3.32	GSM2806227_r1			
1475894	SRR6145429	SRP119561	SRS2570981	SRX3257520	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806228: 16-14T.RNA.Old; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Old|tissue;;lateral temporal lobe	GEO Accession;;GSM2806228		GSM2806228	16-14T.RNA.Old	2964373607	40036785	2018-03-08 00:04:46	1329590340	2964373607	40036785	2	40036785	index:0,count:40036785,average:74.04,stdev:3.83	GSM2806228_r1			
1476119	SRR6145430	SRP119561	SRS2570982	SRX3257521	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806229: 17-9A.RNA.Old; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Old|tissue;;lateral temporal lobe	GEO Accession;;GSM2806229		GSM2806229	17-9A.RNA.Old	2945519445	39919539	2018-03-08 00:04:46	1321108359	2945519445	39919539	2	39919539	index:0,count:39919539,average:73.79,stdev:4.22	GSM2806229_r1			
1476151	SRR6145431	SRP119561	SRS2570983	SRX3257522	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806230: 18-10A.RNA.Old; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Old|tissue;;lateral temporal lobe	GEO Accession;;GSM2806230		GSM2806230	18-10A.RNA.Old	2744308144	36990438	2018-03-08 00:04:46	1231592805	2744308144	36990438	2	36990438	index:0,count:36990438,average:74.19,stdev:3.23	GSM2806230_r1			
1476182	SRR6145432	SRP119561	SRS2570985	SRX3257523	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806231: 19-11A.RNA.Old; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Old|tissue;;lateral temporal lobe	GEO Accession;;GSM2806231		GSM2806231	19-11A.RNA.Old	3253247544	43899259	2018-03-08 00:04:46	1461574482	3253247544	43899259	2	43899259	index:0,count:43899259,average:74.11,stdev:3.62	GSM2806231_r1			
1476214	SRR6145433	SRP119561	SRS2570984	SRX3257524	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806232: 20-1T.RNA.AD; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Aged, diseased|tissue;;lateral temporal lobe	GEO Accession;;GSM2806232		GSM2806232	20-1T.RNA.AD	2938374823	40299174	2018-03-08 00:04:46	1317078485	2938374823	40299174	2	40299174	index:0,count:40299174,average:72.91,stdev:5.62	GSM2806232_r1			
1476245	SRR6145434	SRP119561	SRS2570986	SRX3257525	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806233: 21-1A.RNA.AD; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Aged, diseased|tissue;;lateral temporal lobe	GEO Accession;;GSM2806233		GSM2806233	21-1A.RNA.AD	2941944893	39739040	2018-03-08 00:04:46	1311418158	2941944893	39739040	2	39739040	index:0,count:39739040,average:74.03,stdev:3.77	GSM2806233_r1			
1476278	SRR6145435	SRP119561	SRS2570987	SRX3257526	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806234: 22-2T.RNA.AD; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Aged, diseased|tissue;;lateral temporal lobe	GEO Accession;;GSM2806234		GSM2806234	22-2T.RNA.AD	2918560344	39459711	2018-03-08 00:04:46	1302962050	2918560344	39459711	2	39459711	index:0,count:39459711,average:73.96,stdev:3.86	GSM2806234_r1			
1476311	SRR6145436	SRP119561	SRS2570988	SRX3257527	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806235: 23-2A.RNA.AD; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Aged, diseased|tissue;;lateral temporal lobe	GEO Accession;;GSM2806235		GSM2806235	23-2A.RNA.AD	2820272276	38019285	2018-03-08 00:04:46	1280589911	2820272276	38019285	2	38019285	index:0,count:38019285,average:74.18,stdev:3.34	GSM2806235_r1			
1476344	SRR6145437	SRP119561	SRS2570989	SRX3257528	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806236: 24-3T.RNA.AD; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Aged, diseased|tissue;;lateral temporal lobe	GEO Accession;;GSM2806236		GSM2806236	24-3T.RNA.AD	3020497976	40847873	2018-03-08 00:04:46	1356813114	3020497976	40847873	2	40847873	index:0,count:40847873,average:73.95,stdev:3.88	GSM2806236_r1			
1476375	SRR6145438	SRP119561	SRS2570990	SRX3257529	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806237: 25-5T.RNA.AD; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Aged, diseased|tissue;;lateral temporal lobe	GEO Accession;;GSM2806237		GSM2806237	25-5T.RNA.AD	2626005799	35468886	2018-03-08 00:04:46	1197213689	2626005799	35468886	2	35468886	index:0,count:35468886,average:74.04,stdev:3.20	GSM2806237_r1			
1476406	SRR6145439	SRP119561	SRS2570991	SRX3257530	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806238: 26-3A.RNA.AD; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Aged, diseased|tissue;;lateral temporal lobe	GEO Accession;;GSM2806238		GSM2806238	26-3A.RNA.AD	2704167309	36432788	2018-03-08 00:04:46	1229139985	2704167309	36432788	2	36432788	index:0,count:36432788,average:74.22,stdev:3.07	GSM2806238_r1			
1476628	SRR6145440	SRP119561	SRS2570992	SRX3257531	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806239: 27-5A.RNA.AD; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Aged, diseased|tissue;;lateral temporal lobe	GEO Accession;;GSM2806239		GSM2806239	27-5A.RNA.AD	2838283663	38367345	2018-03-08 00:04:46	1270879150	2838283663	38367345	2	38367345	index:0,count:38367345,average:73.98,stdev:3.92	GSM2806239_r1			
1476662	SRR6145441	SRP119561	SRS2570993	SRX3257532	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806240: 28-8T.RNA.AD; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Aged, diseased|tissue;;lateral temporal lobe	GEO Accession;;GSM2806240		GSM2806240	28-8T.RNA.AD	2368193490	31893808	2018-03-08 00:04:46	1076973809	2368193490	31893808	2	31893808	index:0,count:31893808,average:74.25,stdev:2.70	GSM2806240_r1			
1476693	SRR6145442	SRP119561	SRS2570994	SRX3257533	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806241: 29-6T.RNA.AD; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Aged, diseased|tissue;;lateral temporal lobe	GEO Accession;;GSM2806241		GSM2806241	29-6T.RNA.AD	2681122728	36151808	2018-03-08 00:04:46	1216609368	2681122728	36151808	2	36151808	index:0,count:36151808,average:74.16,stdev:3.00	GSM2806241_r1			
1476726	SRR6145443	SRP119561	SRS2570996	SRX3257534	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806242: 30-9T.RNA.AD; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Aged, diseased|tissue;;lateral temporal lobe	GEO Accession;;GSM2806242		GSM2806242	30-9T.RNA.AD	2967585483	40063230	2018-03-08 00:04:46	1317526652	2967585483	40063230	2	40063230	index:0,count:40063230,average:74.07,stdev:3.61	GSM2806242_r1			
1476758	SRR6145444	SRP119561	SRS2570995	SRX3257535	SRA617775	GEO		Dysregulation of the epigenetic landscape of normal aging in Alzheimer''s disease [RNA-Seq]	Aging is the strongest risk factor for Alzheimer''s disease (AD), although the underlying mechanisms remain unclear. The chromatin state, in particular through the mark H4K16ac, has been implicated in aging and thus may play a pivotal role in age-associated neurodegeneration. Here we compared the genome-wide enrichment of H4K16ac in the lateral temporal lobe of AD individuals against both younger and elderly cognitively normal controls. We find that while normal aging leads to H4K16ac enrichment, AD entails dramatic losses of H4K16ac in the proximity of genes linked to aging and AD. Our analysis highlights the presence of three classes of AD-related changes having distinctive functional roles. Furthermore, we discover an association between the genomic locations of significant H4K16ac changes with genetic variants (SNPs) identified in prior AD genome-wide association studies (GWAS) and with expression quantitative trait loci (eQTLs). Our results establish the basis for an epigenetic link between aging and AD. Overall design: RNA-seq in healthy and diseased human brains; using 8 young healthy brains (Young), 10 aged healthy brains (Old), and 12 aged diseased brains (AD)		GSM2806243: 31-7T.RNA.AD; Homo sapiens; RNA-Seq				RNA-Seq	TRANSCRIPTOMIC	cDNA	single			Total RNA was isolated from 100 mg brain tissue by TRIzol solubilization and phenol-chloroform extraction followed by ethanol precipitation.  Purified RNA was subsequently treated with RNase-free DNase (Qiagen). RNA sequencing libraries were prepared as previously described with a few changes, such as omission of the polyA+ selection step and addition of a double-stranded nuclease (DSN)-mediated step to deplete highly abundant RNA species, such as ribosomal RNA. Briefly, purified RNA was fragmented using RNA Fragmentation Reagent (Thermo Fisher) for 5 min at 70°C and followed by addition of a 5’ phosphate by treating with T4 polynucleotide kinase (NEB). The RNA fragments were then size-selected to 60-250 bp on a 15% TBE-urea gel. Illumina v1.5 RNA adapters were ligated to the 3’ and 5’ ends of the fragmented products using T4 RNA ligase 2, truncated (NEB) and T4 RNA ligase (Promega), respectively. cDNA libraries were generated using Superscript II (Invitrogen) and PCR-amplified using Phusion high-fidelity DNA polymerase (NEB). Libraries were purified on a 6% TBE gel (Invitrogen), treated with DSN and PCR re-amplified. Quality of the resulting sequencing libraries were verified by cloning into Zero Blunt TOPO (Invitrogen) and analyzing 6-10 clones by standard Sanger sequencing to determine the presence of adapters, appropriate size selection, abundance of ribosomal RNA, and the diversity of the library products. The libraries were pooled and sequenced on an Illumina NextSeq 500 using the standard protocols for 75 bp single-read sequencing.	NextSeq 500	source_name;;lateral temporal lobe|study group;;Aged, diseased|tissue;;lateral temporal lobe	GEO Accession;;GSM2806243		GSM2806243	31-7T.RNA.AD	2946541065	39847390	2018-03-08 00:04:46	1320272376	2946541065	39847390	2	39847390	index:0,count:39847390,average:73.95,stdev:4.05	GSM2806243_r1			
